test/test_parse_labels.bats

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bats
+
+setup() {
+    TEST_BREW_PREFIX="$(brew --prefix)"
+    load "${TEST_BREW_PREFIX}/lib/bats-support/load.bash"
+    load "${TEST_BREW_PREFIX}/lib/bats-assert/load.bash"
+
+    DIR="$( cd "$( dirname "$BATS_TEST_FILENAME" )" >/dev/null 2>&1 && pwd )"
```

Comment:
I'm not sure if macOS has the `readlink` command, but on Linux this will canonicalize your directory name without any `cd` / `pwd` shenanigans.

```suggestion
    DIR="$( readlink -e "$(dirname "$BATS_TEST_FILENAME")"  )"
```